### PR TITLE
Bankaccount

### PR DESCRIPTION
**Pull request by darth96**
let me know if im missing anything

### CODE REVIEW

**Comment by darth96** [comment]
hey

File: src/test/java/bankaccount/accounts.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        System.out.println("PLease enter last 4digits account number you would like to deposit from");
+        int last_4= scanner.nextInt();
+        if (last_4==checkingAccountNum || last_4==savingsAccountNum) {
+            System.out.println("Enter the amount you would like to deposit");
+            double amount = scanner.nextDouble();
+            if (amount > 0) {
+                if (last_4 == checkingAccountNum) {
+                    checkingAccountBalance += amount;
+                } else if (last_4 == savingsAccountNum)
+                    checkingAccountBalance += amount;
```
Comment (ahsansiddiki) [inline src/test/java/bankaccount/accounts.java:28]: I think you intended to write ** savingsAccountBalance** in else if block.

File: src/test/java/bankaccount/accounts.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    }
+
+    //method to withdraw
+    public void withdraw(){
+        System.out.println("Please enter last 4 digits account number you would like to withdraw from");
+        int last_4= scanner.nextInt();
+        if (last_4==checkingAccountNum || last_4==savingsAccountNum);{
+        System.out.println("Enter the amount you would like to withdraw");
+         double amount= scanner.nextDouble();
+        if (amount<=checkingAccountNum || amount<=savingsAccountNum){
```
Comment (ahsansiddiki) [inline src/test/java/bankaccount/accounts.java:41]: I believe you wanted to write savingsAccountBalance and checkingAccountBalance in this line.

File: src/test/java/bankaccount/customer.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    //method to edit customer details
+    public void editDetails(){
+        System.out.println("which field you would like to update");
+        System.out.println("A: name");
+        System.out.println("B: address");
+        System.out.println("Please choose A or B");
+        String edit = scanner.next();
+        switch (edit.toUpperCase()){
+            case"A":
+                System.out.println("your name has been changed" + name);
```
Comment (ahsansiddiki) [inline src/test/java/bankaccount/customer.java:26]: Looks like you didn't asked to user to provide new name to update existing name.

**Comment by ahsansiddiki** [comment]
I added few comments on the PR.


Warm Regards

Ahsan Siddiki
(929)300-6788

File: src/test/java/bankaccount/accounts.java (hunk 1 above)
Comment (darth96) [reply src/test/java/bankaccount/accounts.java:28]: Yes that was a mistake

File: src/test/java/bankaccount/customer.java (hunk 3 above)
Comment (darth96) [reply src/test/java/bankaccount/customer.java:26]: Yes forgot about that

File: src/test/java/bankaccount/accounts.java (hunk 2 above)
Comment (darth96) [reply src/test/java/bankaccount/accounts.java:41]: Right

**Comment by darth96** [comment]
Beside those 2 things you mentioned above, that's all right?

**Comment by darth96** [comment]
I fixed those issues

**Pull request closed without merging**